tests/acceptance/book/UpdateBookInfoCest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use \Codeception\Util\HttpCode;
+use Page\Acceptance\BookInfo;
+use Page\Acceptance\CreateBook;
+
+class UpdateBookInfoCest
+{
+	public $bookURL = 'testbookinfo';
+
+	public $bookTitle = 'Test Book Info';
+
+	public $saveButton = '#submitpost input[type=submit]';
```

Comment:
Would it be a good idea to create a Page for the 'edit book information' page just like the CreateBookPart so it can improve reusability and decrease long test in a single file, as there are many fields in the book info page? The current method is straightforward and clear. Nice.